core/lib/refinery/core/engine.rb

```diff
@@ -26,6 +26,35 @@ def refinery_inclusion!
         end
       end
 
+      def self.frontend_routes(&block)
+        @frontend_routes ||= []
+        unless @frontend_routes.include?(block)
```

Comment:
`if @frontend_routes.exclude?(block)` reads easier.

Perhaps we should rename the variable from `block` which is a Ruby notion (i.e. `do .. end` is a block, but we're meaning something else.

Also does `@frontend_routes` really need to be an instance variable? We're accessing a class method `self.frontend_routes` so it's a little bit worrying that in `self.draw_routes` which is also a class method that we're using instance variables.  Perhaps I'm missing something.